plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/shipping-rates-control-package/package-rates.tsx

```diff
@@ -64,6 +64,14 @@ const PackageRates = ( {
 		}
 	}, [ onSelectRate, selectedOption ] );
 
+	// Update the selected option if there is no rate selected on mount.
+	useEffect( () => {
+		if ( ! selectedOption && rates.length > 0 ) {
+			setSelectedOption( rates[ 0 ].rate_id );
+			onSelectRate( rates[ 0 ].rate_id );
+		}
+	}, [ onSelectRate, rates, selectedOption ] );
```

Comment:
this works. 

I'd actually move this one before the useEffect on line 61. Although it makes no difference, at least chronologically this is how events are triggered. 

I'd also add a check on line 46. `return rates[ 0 ]?.rate_id;`, to return the first rate only when `rates.length > 0`; this would make the code more clear and explain why we need this extra effect